### DISCUSSION TITLE
Community meetup?

### DISCUSSION THREAD

**Discussion by rkdarst** [jupyter/nbgrader#1635] [General]
Hi all.

Would anyone be interested in some sort of community event, to get a picture of all the different ways nbgrader is used and developed now?  I have a feeling that a lot has been done in the last few years, but it has been local, without as much sharing as there could be.

I don't think it should be anything big, but this is my initial proposal:
* Online (we need to make the first one as accessible as possible for many people)
* One or two half days
* Two main program ideas:
  * "cool things and problems" presentation: everyone presents a few cool things they have done with nbgrader lately, and a few problems they are working on
  * Self-organized discussion (based on the first part, we will see many problems and solutions to match up)
* Most of the program is made by people writing suggestions in wiki pages/hackmd and then self-organization close to the event itself ("unconference")

I hope the outcomes would include seeing what each other are up to, co-work on making some of our developments reusable, and a plan for future community meetups and development. 

I happened to chat with @gollington, and we could get some sort of Jupyter support for this.

**Comment by damianavila** [comment]
I think this is actually a great general idea! And the proposal outlined above seems reasonable (and interesting!) to me.

**Comment by damianavila** [comment]
It would be great if we can also invite/involve people who are porting nbgrader to jlab.
@SylvainCorlay might have some info about this one? Ref #1006

**Comment by perllaghu** [comment]
There's a few people working on nbgrader to jlab.... and we're one of them
(Our primary challenge is trying to get any existing work to run in jlab 3.3....)


Note that nbgrader works just fine as a command-line tool - it's only the UI stuff that changes [the client-side has moved from javascript to typescript, and there's a whole suite of methods for interacting with the Interface]

**Comment by damianavila** [comment]
> There's a few people working on nbgrader to jlab.... and we're one of them

It would be great if those teams have cross-talks about the work being done so there is no duplication of work 😉.

**Comment by jtpio** [comment]
Maybe the Notebook weekly calls could be a good place to start these community meetings?

They happen on Wednesdays at 08:00 Pacific Time: https://github.com/jupyter/notebook-team-compass#weekly-team-meetings

cc @brichet @SylvainCorlay @fcollonval who might be interested

**Comment by jhamrick** [comment]
Agree this would be a great thing to do!

I probably couldn't join in the near future (I am currently on parental leave 😄) but it would be great to see something like this happen, and if there's a meetup later in the year perhaps I could join then!

**Comment by SylvainCorlay** [comment]
I submitted a proposal for a Jupyter Community Workshop on the usage of Jupyter in education, if this gets accepted, we should definitely move forward with organizing an in-person event.
